introduction.md

```diff
@@ -1,4 +1,3 @@
-I like developing.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?